functorch/experimental/_map.py

```diff
@@ -31,7 +31,7 @@ def _unwrap_proxy(e):
         raise ValueError("map() must loop over a tensor")
     if len(xs.shape) == 0 or xs.shape[0] == 0:
         raise ValueError("map() cannot be traced with scalar tensors or zero dimension tensors")
-    if not all(isinstance(o, (torch.Tensor, torch.nn.Module)) for o in args):
```

Comment:
Why is torch.nn.Module not allowed?

Reply:
hmmm long story short it's not a reasonable contract we put module as graph arguments in the graph for backend.